src/providers/postgres/qgspostgresconn.cpp

```diff
@@ -110,7 +110,7 @@ QString QgsPostgresResult::PQfname( int col )
   return QString::fromUtf8( ::PQfname( mRes, col ) );
 }
 
-int QgsPostgresResult::PQftable( int col )
+unsigned int QgsPostgresResult::PQftable( int col )
```

Comment:
in some part of the master PR you used unsigned int instead of Oid... what's the reason?